app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -64,16 +67,20 @@
     public boolean updatingPreviewViewport = false;
     public boolean updatingChartViewport = false;
     boolean isBTWixel;
+    boolean isDexbridgeWixel;
     boolean isBTShare;
     boolean isWifiWixel;
 
     public BgGraphBuilder bgGraphBuilder;
     BroadcastReceiver _broadcastReceiver;
     BroadcastReceiver newDataReceiver;
+    BroadcastReceiver newSavedBgReceiver;
+    private static Context mContext;
```

Comment:
mContext is not defined prior to the Home activity being shown, but gets referenced in getContext() and gets used in services.  If the Activity is reaped for memory pressure, the services may be restarted and the context will be invalid.  Services are their own context.  Maybe it is more appropriate to initialize BgSendQueue from the services using them or pass in a PebbleSync that was initialized with the service as a context.  Holding a reference to an application Context isn't likely to work in all circumstances.

In most cases these static-rich classes look like good candidates for a bound service and an AIDL interface.  It gets a little hairy writing the first one, but it's pretty simple and provides an API across process boundaries with well-defined contexts, etc for all of them.